src/main/java/com/iemr/tm/config/SwaggerConfig.java

```diff
@@ -24,27 +24,22 @@
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 
-import springfox.documentation.builders.RequestHandlerSelectors;
-import springfox.documentation.service.ApiInfo;
-import springfox.documentation.service.Contact;
-import springfox.documentation.spi.DocumentationType;
-import springfox.documentation.spring.web.plugins.Docket;
-import springfox.documentation.swagger2.annotations.EnableSwagger2;
 
-@Configuration
-@EnableSwagger2
-public class SwaggerConfig {
-	@Bean
-	public Docket productApi() {
-		return new Docket(DocumentationType.SWAGGER_2).select()
-				.apis(RequestHandlerSelectors.basePackage("com.iemr.tm.controller"))
-				.build().apiInfo(metaData());
-	}
 
-	private ApiInfo metaData() {
-		ApiInfo apiInfo = new ApiInfo("Scheduler API", "An interface between the client and scheduling services which allows users to consult with specialists enabling them to book or cancel slots based on availability.",
-				"1.0", "Terms of service", new Contact("AMRIT", "https://psmri.github.io/PSMRI/", "amrit@piramalswasthya.org"), "",
-				"");
-		return apiInfo;
-	}
+//@Configuration
+//@EnableSwagger2
+public class SwaggerConfig {
```

Comment:
Please confirm why this is removed. It's required for generating Swagger JSON AFAIK.